### PR TITLE
[kerbal] update to v2024.11.1

### PR DESCRIPTION
**Pull request by WentsingNee**
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md).
- [x] SHA512s are updated for each updated download.
- [x] The "supports" clause reflects platforms that may be fixed by this new version.
- [x] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

File: ports/kerbal/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "kerbal",
-  "version": "2024.8.1",
+  "version": "2024.11.1",
   "description": "Kerbal C++ Template Library",
   "homepage": "https://github.com/WentsingNee/Kerbal",
   "license": "LGPL-3.0",
```
Comment (jimwang118) [inline ports/kerbal/vcpkg.json:6]:
```suggestion
  "license": "LGPL-3.0-or-later",
```

**Review by jimwang118 — CHANGES_REQUESTED** [review]

**Review by jimwang118 — DISMISSED** [review]

**Comment by jimwang118** [comment]
> @jimwang118 should I squash the second commit into the first one?

You need to run the command: `./vcpkg x-add-version {portname} --overwrite-version`, and then submit the regenerated kerbal.json file.

**Comment by WentsingNee** [comment]
> > @jimwang118 should I squash the second commit into the first one?
> 
> You need to run the command: `./vcpkg x-add-version {portname} --overwrite-version`, and then submit the regenerated kerbal.json file.

done

File: ports/kerbal/vcpkg.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
 {
   "name": "kerbal",
-  "version": "2024.8.1",
+  "version": "2024.11.1",
   "description": "Kerbal C++ Template Library",
   "homepage": "https://github.com/WentsingNee/Kerbal",
-  "license": "LGPL-3.0",
+  "license": "LGPL-3.0-or-later",
```
Comment (BillyONeal) [inline ports/kerbal/vcpkg.json:6]: Can you explain where the `-or-later` is documented? The LICENSE does not say that.

Comment (WentsingNee) [reply ports/kerbal/vcpkg.json:6]: Well, this change is suggested by @jimwang118  during his review. Actually, I didn't change my project's LICENSE these days.

Comment (WentsingNee) [reply ports/kerbal/vcpkg.json:6]: https://github.com/microsoft/vcpkg/pull/42307#discussion_r1853448738

Comment (WentsingNee) [reply ports/kerbal/vcpkg.json:6]: I didn't focus and track the policy published by the vcpkg official so I also don't understand. Whether you have updated the policy related to the LICENSE in pursuit of the compatibility?

Comment (jimwang118) [reply ports/kerbal/vcpkg.json:6]: There is no separate LGPL-3.0 license in the website [SPDX-license-list](https://spdx.github.io/spdx-spec/v2.3/SPDX-license-list/) where we are verifying the license, so it is changed to [LGPL-3.0-or-later](https://spdx.org/licenses/LGPL-3.0-or-later.html).

Comment (dg0yt) [reply ports/kerbal/vcpkg.json:6]: Either it is `LGPL-3.0-or-later` or it is `LGPL-3.0-only`. If we do not know exactly, the deprecated fuzzy `LGPL-3.0` might be the best fit!

Comment (BillyONeal) [reply ports/kerbal/vcpkg.json:6]: @WentsingNee Ah, I'm sorry, I didn't notice that you are upstream! In that case, I agree with @dg0yt that the deprecated fuzzy one should remain. I would suggest that, as the upstream maintainer, you clarify in your own project's documentation which you mean, and we'll update the license expression when we ingest that update. However, we don't need that to happen to take this update. Thanks!

**Review by BillyONeal — APPROVED** [review]
I reverted the license field edit and set auto merge, I assumed @WentsingNee is happy with that given that their initial PR did not contain that change.

**Review by jimwang118 — APPROVED** [review]

**Pull request merged**